lib/__tests__/harvest.test.ts

```diff
@@ -124,16 +124,38 @@ describe("Integration tests", () => {
       it("should retrieve a queryID on page load", async () => {
         expect(data).toHaveProperty("queryID");
       });
-      it("should generate an anonymous userToken on init", async () => {
-        const userToken = await page.evaluate(() => {
-          return new Promise((resolve, reject) =>
-            window.aa("getUserToken", null, (err, res) => {
-              if (err) return reject(err);
-              resolve(res);
-            })
-          );
-        });
+      it("should generate an anonymous userToken on init and store it in a cookie", async () => {
+        const userToken = await page.evaluate(
+          () =>
+            new Promise((resolve, reject) =>
+              window.aa("getUserToken", null, (err, res) => {
+                if (err) return reject(err);
+                resolve(res);
+              })
+            )
+        );
+
+        const cookies = await page.cookies();
+        const algoliaCookie = cookies.find(
+          cookie => cookie.name === "_ALGOLIA"
```

Comment:
Unrelated to this PR but do you have context about why the cookie that stores the user token is called `_ALGOLIA`?

Reply:
no, seems like it was there from day one.
https://github.com/algolia/search-insights.js/commit/8cac6358ac667e06933bb8a10c03e43250cccc13#diff-2bc69471968392e291c0bd99e6836838R3
Do you find the name confusing?

Reply:
Since it's a generic name I expected it to store an object, not only the user token string. But there might be a reason.

Reply:
I agree. I think it was inspired by [Google Analytics' `_ga`](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#cookieName).